contrib/epee/include/net/http_client.h

```diff
@@ -385,11 +385,9 @@ using namespace std;
 
 					req_buff += "\r\n";
 					//--
-
-					bool res = m_net_client.send(req_buff, timeout);
-					CHECK_AND_ASSERT_MES(res, false, "HTTP_CLIENT: Failed to SEND");
 					if(body.size())
-						res = m_net_client.send(body, timeout);
+						req_buff += body;
+					bool res = m_net_client.send(req_buff, timeout);
```

Comment:
Is there a good reason for splitting header and body into separate writes? If not, this change seems to cause less troubles with the REST server used by OpenMonero (restbed). 
@moneroexamples @moneromooo-monero @vtnerd